### PR TITLE
Minor updates to OLM and OperatorHub pages

### PR DESCRIPTION
**Pull request by spadgett**
* Make Subscriptions first tab under Operator Management
* Add help text to top of Subscriptions page
* Change modal title to "Remove Operator Subscription"
* Update help text on Create Operator Subscription page

/assign @rhamilto 
cc @alecmerdler @tlwu2013 @jwforres 

localhost_9000_

localhost_9000_ (1)

localhost_9000_ (2)

### CODE REVIEW

**Comment by spadgett** [comment]
/hold cancel

@rhamilto I had to update the create link URL for subscriptions due to the tab changes. Another look?

**Comment by spadgett** [comment]
performance test is passing locally for me

/retest

File: frontend/public/components/operator-management.tsx [hunk 1]
```diff
@@ -6,12 +6,12 @@ import {InstallPlansPage} from './operator-lifecycle-manager/install-plan';
...
 const pages = [{
   href: '',
-  name: 'Operator Catalogs',
-  component: PackageManifestsPage,
-}, {
-  href: 'subscriptions',
   name: 'Operator Subscriptions',
   component: SubscriptionsPage,
+}, {
+  href: 'catalogs',
```
Comment (alecmerdler) [inline frontend/public/components/operator-management.tsx]: Should be `catalogsources`.

**Comment by spadgett** [comment]
@alecmerdler Thanks, updated.

Seeing performance test failures in CI that I haven't been able to reproduce :/

**Comment by spadgett** [comment]
I fixed the performance tests. This should be ready.

@alecmerdler or @rhamilto another look?

**Comment by spadgett** [comment]
annotations flake

/retest

**Comment by spadgett** [comment]
> StaleElementReferenceError: stale element reference: element is not attached to the page document

/retest

**Pull request merged**